Library/Formula/libodfgen.rb

```diff
@@ -1,5 +1,5 @@
 class Libodfgen < Formula
-  homepage "http://sourceforge.net/p/libwpd/wiki/libodfgen/"
+  homepage "https://sourceforge.net/p/libwpd/wiki/libodfgen/"
```

Comment:
This _was_ the case indeed, but it seems to stay SSL/TLS now even when not being logged in. At least that's what happens in my browser since a few weeks and now with this URL.